### ISSUE TITLE
Название функции get_root_window не соответствует содержанию

### ISSUE THREAD

**Issue by Mingun** [AndrewCrewKuznetsov/xneur-devel#47]
Функция не самом деле возвращает родительское окно, а не корневое, хотя имя говорит, что возвращает оно корневое. Кроме того, непонятна проверка `root_window == parent_window`. Т.е. на самом деле возвращается не корневое окно, а его непосредственный потомок:
https://github.com/AndrewCrewKuznetsov/xneur-devel/blob/362019ac27f388b319b654d33ec3b1799339bf72/xneur/lib/main/utils.c#L62-L77

Возможно, здесь ошибка. Нужно или исправить ее, или назвать функцию корректно, желательно также описать, что она должна делать.

**Comment by Genues** [comment]
Очень хрошо, что кто-то взялся за доведения этой очень полезной программы до ума. Т.к. автор похоже совсем на неё забил.

**Comment by Mingun** [comment]
Да, надоело уже что вечно он глючит с буферами обмена и не стирает текст до конца. Если автор не объявится, придется методом тыка разбираться, где здесь ошибки
